evm/test/Coordinator_test.js

```diff
@@ -252,35 +252,42 @@ contract('Coordinator', () => {
         const tx = await link.transferAndCall(
           coordinator.address,
           agreement.payment,
-          payload
+          payload,
+          { value: 0 }
```

Comment:
Do we need this extra set of params? You've specified `value: 0` in a couple of places, but that's also the default. I typically lean towards less code when possible, and if we can just drop the extra options then the parts of the test that matter can be a little more clear.

Reply:
The reason why I did this is that the new version of truffle-contract will submit a value of `undefined` if no options are specified. I'm not sure why this is happening, but adding the (redundant) option of `{ value: 0 }` fixed this issue. I can try to find another workaround, but for the time being I think this may be the best solution.